### ISSUE TITLE
import onnxruntime error

### ISSUE THREAD

**Issue by qiaodalei** [microsoft/onnxruntime#1127]
When I try to import onnxruntime, it return error as below:
ImportError: cannot import name 'RunOptions' from 'onnxruntime.capi._pybind_state' 

I checked the cuda, cudnn version and path, it doesn't work. Finally I find it is the problem of VC runtime. Maybe the error massage could be clearer.

**Comment by pranavsharma** [comment]
What's the full error? What was the problem of VC runtime? Also, take a look at this https://github.com/Microsoft/onnxruntime#system-requirements. 

For e.g. in my case when I don't have the required library dependencies, it clearly tells me so.
```
(py36) pranav@pranav-HP-Z440-Workstation:~$ python
Python 3.6.8 |Anaconda custom (64-bit)| (default, Dec 30 2018, 01:22:34)
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import onnxruntime
/home/pranav/.conda/envs/py36/lib/python3.6/site-packages/onnxruntime/capi/_pybind_state.py:12: UserWarning: Cannot load onnxruntime.capi. Error: 'libcublas.so.9.1: cannot open shared object file: No such file or directory'
  warnings.warn("Cannot load onnxruntime.capi. Error: '{0}'".format(str(e)))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/pranav/.conda/envs/py36/lib/python3.6/site-packages/onnxruntime/__init__.py", line 21, in <module>
    from onnxruntime.capi._pybind_state import RunOptions, SessionOptions, get_device, NodeArg, ModelMetadata
ImportError: cannot import name 'RunOptions'
```

**Comment by qiaodalei** [comment]
It's the same problem like #251 when I try to import onnxruntime as ort

UserWarning: Cannot load onnxruntime.capi. Error: 'DLL load failed: The specified module cound not be found.'
  warnings.warn("Cannot load onnxruntime.capi. Error: '{0}'".format(str(e)))
  File "C:\Users\xxxx\AppData\Local\Programs\Python\Python37\lib\site-packages\onnxruntime\__init__.py", line 21, in <module>
    from onnxruntime.capi._pybind_state import RunOptions, SessionOptions, get_device, NodeArg, ModelMetadata
ImportError: cannot import name 'RunOptions'

When I got this error, I found the anwser of #251 and I checked cuda and cudnn and path.
Cuda 10.0
cudnn 7.6
python 3.7.3
tf-gpu 1.13
And the path is ok.

And finally, I find that I ignored this:
For Windows, OpenMP support comes as part of VC runtime. It is also available as redist packages: vc_redist.x64.exe and vc_redist.x86.exe
It seems that without this, no metter with  tf or tf-gpu, I can't import onnxruntime, and get same error message. After I installed visual studio 2017, it works with tf and tf-gpu. 

Hope that my solution can help others.
